contracts/multisig-prover/src/contract/execute.rs

```diff
@@ -230,16 +230,28 @@ pub fn update_verifier_set(
                 .save(deps.storage, &new_verifier_set)
                 .map_err(ContractError::from)?;
 
-            Ok(Response::new().add_message(
-                wasm_execute(
-                    config.multisig,
-                    &multisig::msg::ExecuteMsg::RegisterVerifierSet {
-                        verifier_set: new_verifier_set,
-                    },
-                    vec![],
+            let verifier_union_set = all_active_verifiers(&deps)?;
```

Comment:
what's the difference between new_verifier_set?

Reply:
no difference, this function will just end up transforming the type in this case. But maybe not the best because it will read from state again. I'll change it